kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/core/annotation/tags.kt

```diff
@@ -10,3 +10,10 @@ package io.kotest.core.annotation
 @Retention(AnnotationRetention.RUNTIME)
 annotation class Tags(vararg val values: String)
 
+/**
+ * Attach to a [io.kotest.core.spec.Spec], and that spec won't be instantiated if the tag(s) are not
+ * provided at runtime. If more than one tag is specified then all tags must be provided at runtime.
```

Comment:
Nit:

Add to these docs on how to add a tag to a spec (even if just a link)

Reply:
This was just a file move.
Once we have this all merged in I will write up considerably detailed docs on all these changes (and the rest of 6.0